src/client/ABRPublishing.js

```diff
@@ -95,11 +98,17 @@ exports.CreateProductionMaster = async function({
   structLogLevel="none"
 }) {
   ValidateLibrary(libraryId);
+  let id;
+  const finalize = !writeToken;
```

Comment:
are we using this `const finalize`?

Reply:
Yes, towards the end of the function. If truthy, it finalizes the content object.